keyence_library/include/keyence/impl/messages/get_setting.h

```diff
@@ -0,0 +1,49 @@
+#ifndef KEYENCE_GET_SETTING_H
+#define KEYENCE_GET_SETTING_H
+
+#include "../keyence_message.h"
+
+namespace keyence
+{
+namespace command
+{
+
+class GetSetting
+{
+public:
+  // Forward declares
+  struct Request;
+  struct Response;
+
+  struct Request
+  {
+    typedef Response response_type;
+    const static uint32_t size = 16;
+    const static uint8_t command_code = 0x31;
```

Comment:
This is one case where I disagree with the style guide. I'm slightly uncomfortable defining something like `size` as `SIZE` because it could possibly collide with a C MACRO from some other header.

Plus I'd have to change all of the other commands to match. If we want to do that, I'd prefer to address it in another PR.